Makefile

```diff
@@ -1,11 +1,13 @@
 BIN_NAME=terraform-provider-instaclustr
 
-VERSION=2.0.99
+VERSION=2.0.100
 
 FULL_BIN_NAME="${BIN_NAME}_v${VERSION}"
 SHASUM_NAME="${BIN_NAME}_${VERSION}_SHA256SUMS"
+GOOS="darwin"
```

Comment:
Making these into variables to they can be specified as required by the os and arch while running make install